objects/applications/data_hub/dh-psi-monitoring.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Would we still need the `---` if there's only a single object per file?

Reply:
removed